xbmc/video/VideoDatabase.cpp

```diff
@@ -3518,6 +3614,11 @@ CVideoInfoTag CVideoDatabase::GetDetailsForMovie(const dbiplus::sql_record* cons
     }
     m_pDS2->close();
 
+    std::string strSQL = PrepareSQL("SELECT * FROM bookmark JOIN movie ON movie.c%02d=bookmark.idBookmark WHERE movie.idMovie=%i AND bookmark.type=%i", VIDEODB_ID_MOVIE_BOOKMARK, details.m_iDbId, CBookmark::MOVIE);
```

Comment:
Only SELECT the value that you need, i.e. `bookmark.timeInSeconds` and then use `m_pDS2->fv(0).get_asFloat()`.